app/models/review.rb

```diff
@@ -7,13 +7,22 @@ class Review < ActiveRecord::Base
   validates :title, length: {
     minimum: 1,
     maximum: 100,
+    too_short: "Titles need to be more than 1 character",
     too_long: "Titles need to be under 100 characters" },
-    allow_blank: true
+    presence: { message: "You wrote a review, but forgot the title."}, if: :body_filled?
   validates :body, length: {
     minimum: 50,
     maximum: 5000,
     too_short: "Reviews need to be at least 50 characters",
     too_long: "Reviews need to be under 5000 characters" },
-    allow_blank: true
+    presence: { message: "You wrote a title, but forgot the review."}, if: :title_filled?
```

Comment:
Line is too long. [89/80]<br>Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.